srv/GetExternalCollisionProtectionEnabled.srv

```diff
@@ -0,0 +1,3 @@
+# Get information about External Collision Protection Enabled
+naoqi_bridge_msgs/ExternalCollisionProtectionNameList name
```

Comment:
you need [] if you want a list. But replace NameList by Names for simplicity I guess.